### DISCUSSION TITLE
trurl and C89

### DISCUSSION THREAD

**Discussion by jacobmealey** [curl/trurl#269] [General]
Should trurl strive to be C89 compatible? Is that something we would want before a 1.0 release?  

@jay layed out some of the work in the PR: (https://github.com/curl/trurl/pull/268)

**Comment by bagder** [comment]
I'm personally quite used to writing C89 code from curl/libcurl, but I am not convinced it is important for trurl. I don't think this tool is likely to get built on platforms without support for C99.

If we want C89, then we should probably make sure a CI job makes sure to use options to complain if we fail to live up to it.
